appserver/security/core-ee/src/main/java/com/sun/enterprise/security/jacc/provider/SimplePolicyConfigurationFactory.java

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
Is it the correct license header if it is a new file?

Reply:
I updated it, thx ;)